### PR TITLE
add enabled_departments

### PR DESCRIPTION
**Pull request by jakedave**
# Description of change
Adds `enabled_departments` as a non-mandatory array

[source](https://developer.zendesk.com/rest_api/docs/chat/agents)

References issue: #26 

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

**Comment by cmerrick** [comment]
Hi @jakedave, thanks for your contribution!

In order for us to evaluate and accept your PR, we ask that you [sign a contribution license agreement](https://stitch-cla-enforcer.herokuapp.com/). It's all electronic and will take just minutes.

**Comment by cmerrick** [comment]
You did it @jakedave!

Thank you for signing the Singer Contribution License Agreement.

**Comment by SeanFLynch** [comment]
Any idea when this will be reviewed and merged?

**Comment by rushi** [comment]
Hi @cmerrick Hope this can be merged soon. We are affected with this issue as well.

**Comment by rushi** [comment]
Hi @cmerrick Hope this can be merged soon. We are affected with this issue as well and it's blocked all replication

**Comment by jakedave** [comment]
@SeanFLynch @rushi I tried reaching out to the repo's maintainers via email.  This repo hasn't been maintained in years though, I'm not optimistic my PR (or any at all) will be merged.

**Comment by hz-lschick** [comment]
I am in favor of this commit. But just an idea to work yourself around this issue until this is merged:

I've published my version of the singer tap here #28 which implements singer transform before writing the records to the target. When you don't have a need for this property, my version of this tap will clear out the property so you should not get any issues anymore.

**Comment by rushi** [comment]
I use StitchData to sync data from Zendesk to Google BigQuery. Ideally I would want @jakedave's change, but I don't think Stitch has a method to add a custom integration of my own. Otherwise I would just get Jake's changes, compile and use that.

**Comment by rushi** [comment]
I use StitchData to sync data from Zendesk to Google BigQuery. Ideally I would want @jakedave's change, but I don't think Stitch has a method to add a custom integration of my own. Otherwise I would just get Jake's changes, compile and use that.

@cmerrick Anyway we can use these changes without waiting it to be merged upstream?

**Comment by hz-lschick** [comment]
@rushi You might be able to push StichData to merge it via the Singer Slack channel: https://singer-slackin.herokuapp.com/
Or you create a supprt request for them

**Review by KAllan357 — APPROVED** [review]

**Pull request merged**
